Makd.mak

```diff
@@ -267,6 +270,9 @@ PKGVERSION := $(shell echo $(VERSION) | \
 # http://www.gnu.org/software/make/manual/make.html#Text-Functions
 TEST_FILTER_OUT := $C/$(SRC)/%/main.d
 
+# Directories to search for files to convert using d1to2fix
+D1TO2FIX_DIRS := $C
```

Comment:
I mean with `:=`. Again, what I'm saying here is we already have other variables that are supposed to be overrideables defined with `:=`.

The dependencies here are complicated, because there are some things that are supposed to be overrideable via Config.mak and other that are not, because they depedend on the things defined in Config.mak to calculate them. Again, I'm not saying using `?=` won't work, I'm just saying that this is a pre-existing problem that to be solved it will need a careful review of all the variables and where they should be overridden, something that I don't want to address in this PR, and why I used the same operator being used in this section.